ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html

```diff
@@ -48,8 +48,8 @@
                         value="${item.toString}"
                         disabled="${!radioButton.enabled || radioButton.readOnly}"
                         checked="${radioButton.enums[itemList.index] == radioButton.default[0]}"
-                        aria-readonly="${radioButton.readOnly ? 'true' : ''}"/>
-                  <span>${radioButton.enumNames[itemList.index]}</span>
+                        aria-describedby="${radioButton.enumNamesAsTextContent[itemList.index].value} ${radioButton.id}__errormessage"/>
```

Comment:
Please wait for shivam's change and then revert this.

Reply:
The 'aria-describedby' attribute only accepts the reference ID of the description or error element

Reply:
Added reference of error element